src/components/Code/Code.scss

```diff
@@ -1,76 +1,74 @@
 @import "./../../base/base.scss";
 @import "../Input/Input.scss";
```

Comment:
The file naming should be consistent

Reply:
any suggestions for the file name ? @Zissan

Reply:
Actually it was regarding the case, base.scss is in lower case, and Input.scss in pascal case.

Reply:
It should be consistent.

Reply:
Okay, we can change this in a separate PR. If I rename Base file then will need to make the import changes in multiple files.